src/Core/System/Language/LanguageDefinition.php

```diff
@@ -104,13 +105,14 @@ public function since(): ?string
 
     protected function defineFields(): FieldCollection
     {
-        $collection = new FieldCollection([
+        return new FieldCollection([
             (new IdField('id', 'id'))->addFlags(new ApiAware(), new PrimaryKey(), new Required()),
             (new ParentFkField(self::class))->addFlags(new ApiAware()),
             (new FkField('locale_id', 'localeId', LocaleDefinition::class))->addFlags(new ApiAware(), new Required()),
             (new FkField('translation_code_id', 'translationCodeId', LocaleDefinition::class))->addFlags(new ApiAware()),
 
             (new StringField('name', 'name'))->addFlags(new ApiAware(), new Required()),
+            (new BoolField('active', 'active'))->addFlags(new ApiAware(), new Required()),
```

Comment:
can you define getDefaults and say `active` is true?

Reply:
🤔 Oh true. In that case, I guess I can remove `new Required()` when it defaults to `false`, right? Just don't want to have language active by default